src/utils/data.js

```diff
@@ -188,8 +215,9 @@ export class DataUtil {
     };
 
     if (this.days > 1) {
-      basalBolusData.basal = basalBolusData.basal / this.days;
-      basalBolusData.bolus = basalBolusData.bolus / this.days;
+      const nDays = this.getNumDaysWithInsulin(bolusData, basalData);
```

Comment:
looking at the code, I think the number of days `nDays` should be included in the basalBolusData so that it can then be displayed in the widget. It may clarify the calculations we do for the end users.